src/components/SearchBar/SearchBar.stories.tsx

```diff
@@ -21,22 +34,7 @@ export default {
       },
     },
     label: {
-      control: {
-        options: ['headline', 'claims', 'Test'],
-        type: 'string',
-      },
-    },
-    size: {
-      control: {
-        options: ['small', 'medium', 'large'],
-        type: 'radio',
-      },
-    },
```

Comment:
Removed to just use the default size, no need to be able to adjust this I think.